lib/data/content/chapter-7.yml

```diff
@@ -22,14 +22,23 @@ case_studies:
     card_source: A farmer in Tanzania © Rafal Cichawa / Adobe Stock
     label: 'Box 10'
     title: 'Governance assessment in Tanzania reveals enormous potential for ‘Territories of Life’'
+<<<<<<< HEAD
```

Comment:
Breaking change here, left in from conflict resolution. Will correct before continuing.